src/classes/project_data.py

```diff
@@ -407,6 +407,58 @@ def rescale_keyframes(self, scale_factor):
         scaled = scaler(copy.deepcopy(self._data))
         return scaled
 
+        # Create copy of active project data
```

Comment:
Is this block of code from a merge? perhaps a regression? I'm thinking this is not supposed to be here. :smile: